R/MsBackendDataFrame-functions.R

```diff
@@ -85,8 +85,6 @@ NULL
     rtime = "numeric",
     acquisitionNum = "integer",
     scanIndex = "integer",
-    mz = "NumericList",
```

Comment:
@lgatto what was the reason to remove `mz` and `intensity` here? I think that removing these lines will cause some downstream problems. Also, what puzzled me is that this seems to be present in the Bioconductor master branch but not (yet) in the github branch?